src/sentry/tasks/relay.py

```diff
@@ -87,6 +93,11 @@ def schedule_build_project_config(public_key):
         public_key=public_key, project_id=None, organization_id=None
     )
 
+    # TODO: Temprorary task
+    check_build_project_config.apply_async(
+        kwargs={"public_key": public_key, "tmp_scheduled": tmp_scheduled}, countdown=7
```

Comment:
do celery workers just not keep these in memory till they're ready to run?  I fear otherwise we have to go through the path of creating not just a new queue (easy) but also getting dedicated workers for it which seems a bit over the top.

Reply:
now moved queue which has it's own workers